src/location/location.resource.test.tsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import useSWR from "swr";
+import { openmrsFetch } from "@openmrs/esm-framework";
+import { useLoginLocations } from "./location.resource";
+import { LoginLocation } from "../types";
+
+jest.mock("@openmrs/esm-framework", () => {
+  const originalModule = jest.requireActual("@openmrs/esm-framework");
+  return {
+    __esModule: true,
+    ...originalModule,
+    openmrsFetch: jest.fn(() => "mocked fetch"),
+  };
+});
+
+jest.mock("swr");
+
+describe("Location Resource tests", () => {
+  test("useLoginLocations should call proper endpoint via SWR", () => {
+    // @ts-ignore
+    useSWR.mockImplementation(() => ({
+      data: { data: "mockedLoginLocations" },
+    }));
+    const loginLocations = useLoginLocations();
+    expect(useSWR).toHaveBeenCalledWith(
+      "/ws/fhir2/R4/Location?_tag=login%20location&_count=100",
+      openmrsFetch
+    );
+  });
+  test("useLoginLocations should parse into Login Locations Array", () => {
+    // @ts-ignore
+    const queryResultsBundle = {
```

Comment:
i think we can move this to separate mock file so that we don't have a lot of these in the test file